infrastructure/gsuite-exporter/pylintrc

```diff
@@ -0,0 +1,313 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
Should this have a different license than the rest of the files?

Reply:
Yes, I copied this file from Forseti by mistake. Removing the file for now (we'll need one later when we add tests)